packages/ckeditor5-special-characters/docs/features/special-characters.md

```diff
@@ -79,11 +79,10 @@ import SpecialCharacters from '@ckeditor/ckeditor5-special-characters/src/specia
 import SpecialCharactersEssentials from '@ckeditor/ckeditor5-special-characters/src/specialcharactersessentials';
 
 function SpecialCharactersArrowsExtended( editor ) {
```

Comment:
The name of this function does not tell what it does after updating its body. It could be renamed to `SpecialCharactersExtend`.

Reply:
The same applies to `packages/ckeditor5-special-characters/docs/_snippets/features/special-characters-extended-category.js`.

Reply:
The function name is incorrect, good catch. @DawidKossowski, could you rename it to `SpecialCharactersExtended`?

You don't need to create another issue, create a branch `ck/6167-update-docs` and make a PR from that branch.

Reply:
Done, #12442